var/spack/repos/builtin/packages/cudnn/package.py

```diff
@@ -23,6 +23,8 @@ class Cudnn(Package):
     maintainers = ['adamjstewart']
 
     # cuDNN 7.6.5
+    version('7.6.5.32-10.2-linux-x64',
+            sha256='600267f2caaed2fd58eb214ba669d8ea35f396a7d19b94822e6b36f9f7088c20')
```

Comment:
We can probably make this version preferred now. Or just have it be random. This is something most users will likely have to manually set for themselves.